.github/workflows/build.yml

```diff
@@ -20,17 +20,17 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [buildjet-4vcpu-ubuntu-2204-arm, buildjet-4vcpu-ubuntu-2204]
+        os: [buildjet-4vcpu-ubuntu-2204-arm, zededa-ubuntu-2204]
```

Comment:
arm64 label needs to be adjusted as well, right?

Reply:
we didn't deploy arm runners, should we?